includes/functions.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
Within WordPress, the `functions.php` file is usually the theme entry point. It's a little odd to have a file within a plugin with this name. Maybe we can considering renaming this file.